Helper: Various Improvements

kovshenin opened the pull request:
Grouped commits for bug fixes and various improvements to the Helper. This is work in progress.

kovshenin committed:
Helper: Improve messaging and CTAs

kovshenin committed:
Helper: Update copy on connect flow

kovshenin committed:
Helper: Improve messaging around plugins without a subscription

josemarques committed:
removes toggle from extensions that cannot be activated

kovshenin committed:
Helper: Add new `WC_Helper_Compat` class for legacy helper compat

This new class removes some of the actions from the legacy helper
(updater) plugin, attempts to migrate an existing legacy connection
to a new one, attempts to deactivate the old helper plugin.

kovshenin committed:
Helper: Improve messaging around maxed out subscriptions

kovshenin committed:
Helper: Enable `WC_Tracker` when connected to WooCommerce.com

josemarques committed:
Helper: adds styles for manual licence form row and other visual improvements

josemarques committed:
Helper: removes disabled toggles

josemarques committed:
Helper: adds styles for connection component

josemarques committed:
Helper: redesigned connection module

josemarques committed:
Helper: rotates the connection module chevron on mouse hover

claudiosanches commented on assets/css/helper.scss:
```diff
@@ -171,6 +171,156 @@ $color_button_secondary: $woo_pink2;
+------------------------------------------------------------------------------*/
+
+.wc-helper {
+	.subscriptions-header{
```
We need apply some coding standards for CSS, getting really messy with a lot of different styles, like here all CSS without space before `{`, something that you don't see in any of the others 15 SCSS files in this project.

josemarques committed:
Helper: adds some fixes to the key form on mobile and other updates to meet coding standards

kovshenin committed:
Helper: Only mark action as primary if there's a CTA button.

kovshenin committed:
Helper: Better handling for woothemes_updater_notice

Remove the updater notice built into extensions and themes, even
if the legacy Helper plugin is deactivated.

kovshenin committed:
Helper: Navigate existing Helper users to the new place.

Adds a Dashboard menu item and direct users of the old helper
plugin to the new Extensions screen.

kovshenin committed:
Helper: Do not show the back-compat menu item for new users

kovshenin committed:
Helper: Avoid notice in API authentication if credentials are missing

kovshenin committed:
Helper: Hide the back-compat menu item when users see the new screen

Also sets the user_id for the connection if one was not previously set.

kovshenin committed:
Merge remote-tracking branch 'origin' into feature/helper
